.github/dependabot.yml

```diff
@@ -0,0 +1,50 @@
+---
+version: 2
+registries:
+  dockerhub: # Define access for a private registry
+    type: docker-registry
+    url: registry.hub.docker.com
+    username: ${{ secrets.DOCKERHUB_LOGIN }}
+    password: ${{ secrets.DOCKERHUB_PASSWORD }}
+updates:
+  - package-ecosystem: github-actions
+    directory: "/"
+    schedule:
+      interval: weekly
+    commit-message:
+      prefix: github-actions
+      include: scope
+    labels:
+      - github-actions
+      - dependencies
+
+  - package-ecosystem: pip
+    directory: "/"
+    schedule:
+      interval: weekly
+    commit-message:
+      prefix: pip
+      include: scope
+    labels:
+      - pip
+      - dependencies
+
+  - package-ecosystem: docker
+    registries:
+      - dockerhub
+    directory: /
+    schedule:
+      interval: weekly
+    commit-message:
+      prefix: docker
+      include: scope
+    labels:
+      - docker
+      - dependencies
+    groups:
+      docker:
+        patterns:
+          - "*"
+        update-types:
+          - patch
+          - minor
```

Comment:
As I'm still experimenting with groups (per example I think I'm facing an issue with pip on the runner-manager), let's remove it for now.
```suggestion

```